kubernetes/services/sg_manager.yaml

```diff
@@ -56,7 +58,7 @@ spec:
   ports:
     - port: 9008
       targetPort: 8080
-      nodePort: 30010
+      nodePort: 30011
```

Comment:
@kevin-zhonghao a general recommendation is to keep the last digit of nodePort same as port, for example, 30008 for sg manager. This would help in the debugging process.

Reply:
Sure, made it